.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
The `launch.json` and `task.json` files are specific to the executor so they should be in `/src/executor/.vscode`.